src/sentry/snuba/models.py

```diff
@@ -10,6 +10,9 @@
 from sentry.db.models.manager import BaseManager
 
 
+QUERY_AGGREGATION_DISPLAY = ["events", "users affected"]
```

Comment:
Since this is kind of display logic, it probably just belongs in the slack file.